### PR TITLE
Removed unused functions

### PR DESCRIPTION
**Pull request by bernd-wechner**
After reviewing the API for tagsources it becomes clear that the only thing imported from any is the `info` attribute and that leaves three, seemingly legacy functions, polluting the discogs tagsource code. 

They have been removed, and a runtime test performed to rule out any oversights.

Sorry to be flooding you with PRs. But I am keeping them small, and as independent as possible for very simple review and merge.

### CODE REVIEW

**Comment by sandrotosi** [comment]
> Sorry to be flooding you with PRs. But I am keeping them small, and as independent as possible for very simple review and merge.

no problem at all! I vary much appreciate a stream of easily reviewable PRs than a colossal single one :)

**Comment by sandrotosi** [comment]
@bernd-wechner i'm afraid this PR now has a conflict: can you take care of it? thanks!

**Comment by bernd-wechner** [comment]
Done.

File: puddlestuff/tagsources/discogs.py [hunk 1]
```diff
@@ -183,44 +173,8 @@ def parse_album_json(data):
...
-            SITE_URL + info['discogs_uri'])
-
-        albums.append(capture_dict_values(info))
-
-    return albums
-
-
-def retrieve_album(info, image=LARGEIMAGE, rls_type=None):
+  
+  def retrieve_album(info, image=LARGEIMAGE, rls_type=None):
```
Comment (sandrotosi) [inline puddlestuff/tagsources/discogs.py:177]: @bernd-wechner i think this should be de-dentent 2 spaces?

Comment (bernd-wechner) [reply puddlestuff/tagsources/discogs.py:177]: Indeed it should not be indented. A bungle in the conflict resolution.

Comment (sandrotosi) [reply puddlestuff/tagsources/discogs.py]: all good, thanks for fixing it so quickly!

**Pull request merged**